include/DgmOctree.h

```diff
@@ -1215,8 +1214,39 @@ namespace CCCoreLib
 		//! Std. dev. of cell population per level of subdivision
 		double m_stdDevCellPopulation[MAX_OCTREE_LEVEL + 1];
 
-		//! Multithreading wrapper
-		MultiThreadingWrapper* m_MT_wrapper;
+		//! Octree cell description helper struct
+		struct octreeCellDesc
+		{
+			DgmOctree::CellCode truncatedCode;
+			unsigned i1, i2;
+			unsigned char level;
+		};
+
+		//! Structure containing objects needed to run octree operations in parallel
+		struct MultiThreadingWrapper
```

Comment:
Why do you bring it back to the header file? It has been (voluntarily) moved to the cpp file a while ago 😅